webrender/src/frame.rs

```diff
@@ -128,6 +129,23 @@ impl<'a> FlattenContext<'a> {
             );
         }
 
+        // Draw scrollars for other scroll frames
+        for (clip_id, clip_node) in &self.clip_scroll_tree.nodes {
```

Comment:
I'm thinking in `flatten_root` I should filter the clip scroll nodes by the current PipelineId, else we'll run into trouble with a nested Iframe?

Reply:
Hrm. It seems like it would be a lot easier just to add the item when processing the scroll frame item during flattening. Otherwise I think that stacked scroll frames will have all their scrollbars on the top of the scene.